src/utils/ExtensionLoader.js

```diff
@@ -406,7 +406,6 @@ define(function (require, exports, module) {
                     "QuickOpenJavaScript",
                     "QuickView",
                     "UrlCodeHints",
-                    "WebPlatformDocs",
```

Comment:
Keep this change, and remove all others (i.e., don't delete the files, don't remove trailing whitespace, etc.).  We want the fewest changes to Brackets src as possible.